test/object-store/util/test_file.hpp

```diff
@@ -266,7 +266,7 @@ class TestSyncManager {
 
         realm::util::UniqueFunction<void(const realm::app::Response&)>& network_callback;
     };
-    std::shared_ptr<realm::app::GenericNetworkTransport> transport = std::make_shared<Transport>(network_callback);
+    const std::shared_ptr<realm::app::GenericNetworkTransport> transport;
```

Comment:
Assigning to `transport` after the `TestSyncManager` is constructed doesn't do anything useful and I wasted a bit of time discovering that, so I made it const.